### PR TITLE
Add version check document in PAI, remote, kubeflow and frameworkcontroller

### PR DESCRIPTION
**Pull request by SparkSnail**
1. Add version check document
2. Add try catch in gpu scheduler to fix UT

### CODE REVIEW

File: docs/en_US/RemoteMachineMode.md [hunk 1]
```diff
@@ -63,3 +63,6 @@ nnictl create --config ~/nni/examples/trials/mnist-annotation/config_remote.yml
 ```
 
 to start the experiment.
+
+## version check
+NNI will check the version of NNIManager and trialKeeper, [refer](PAIMode.md)
```
Comment (xuehui1991) [inline docs/en_US/RemoteMachineMode.md]: since version 0.6？

File: docs/en_US/KubeflowMode.md [hunk 2]
```diff
@@ -196,4 +196,7 @@ Notice: In kubeflow mode, NNIManager will start a rest server and listen on a po
 
 Once a trial job is completed, you can goto NNI WebUI's overview page (like http://localhost:8080/oview) to check trial's information. 
 
+## version check
```
Comment (xuehui1991) [inline docs/en_US/KubeflowMode.md]: since version 0.6？

File: docs/en_US/FrameworkControllerMode.md [hunk 3]
```diff
@@ -97,4 +97,7 @@ Trial configuration in frameworkcontroller mode have the following configuration
    * frameworkAttemptCompletionPolicy: the policy to run framework, please refer the [user-manual](https://github.com/Microsoft/frameworkcontroller/blob/master/doc/user-manual.md#frameworkattemptcompletionpolicy) to get the specific information. Users could use the policy to control the pod, for example, if ps does not stop, only worker stops, this completionpolicy could helps stop ps.
 
 ## How to run example
-After you prepare a config file, you could run your experiment by nnictl. The way to start an experiment on frameworkcontroller is similar to kubeflow, please refer the [document](./KubeflowMode.md) for more information.
\ No newline at end of file
+After you prepare a config file, you could run your experiment by nnictl. The way to start an experiment on frameworkcontroller is similar to kubeflow, please refer the [document](./KubeflowMode.md) for more information.
+
+## version check
```
Comment (xuehui1991) [inline docs/en_US/FrameworkControllerMode.md]: since version 0.6？

File: src/nni_manager/training_service/local/gpuScheduler.ts [hunk 4]
```diff
@@ -85,9 +85,13 @@ class GPUScheduler {
 
     public async stop() {
         this.stopping = true;
-        const pid: string = await fs.promises.readFile(path.join(this.gpuMetricCollectorScriptFolder, 'pid'), 'utf8');
-        await cpp.exec(`pkill -P ${pid}`);
-        await cpp.exec(`rm -rf ${this.gpuMetricCollectorScriptFolder}`);
+        try {
+            const pid: string = await fs.promises.readFile(path.join(this.gpuMetricCollectorScriptFolder, 'pid'), 'utf8');
```
Comment (yds05) [inline src/nni_manager/training_service/local/gpuScheduler.ts]: Suggest to consider how to show errors in NNI cleanup to end-user, but you can improve later after v0.6 release.

File: docs/en_US/RemoteMachineMode.md (hunk 1 above)
Comment (SparkSnail) [reply docs/en_US/RemoteMachineMode.md]: added version information.

File: docs/en_US/KubeflowMode.md (hunk 2 above)
Comment (SparkSnail) [reply docs/en_US/KubeflowMode.md]: add version information.

File: docs/en_US/FrameworkControllerMode.md (hunk 3 above)
Comment (SparkSnail) [reply docs/en_US/FrameworkControllerMode.md]: added version information.

File: src/nni_manager/training_service/local/gpuScheduler.ts (hunk 4 above)
Comment (SparkSnail) [reply src/nni_manager/training_service/local/gpuScheduler.ts]: ok, will refactor the logic in next release.

**Pull request merged**